examples/non-fungible/README.md

```diff
@@ -123,6 +123,9 @@ BLOB_HASH=$(echo "$QUERY_RESULT" | jq -r '.publishDataBlob')
     }
 ```
 
+Whenever we run a query or a mutation, the result of it gets stored in a `QUERY_RESULT` bash variable.
```

Comment:
How does it end up in `QUERY_RESULT`? Doesn't the reader usually just put the queries and mutations into a GraphiQL interface in the browser?

Reply:
Good point, this is actually just for the readme tests. I'll clarify it in the comment